scripts/crontab/crontab.tpl

```diff
@@ -40,6 +40,8 @@ HOME=/tmp
 
 # Once per week
 1 9 * * 1 %(django)s review_reports
+1 15 * * 1 %(django)s process_addons --task=constantly_recalculate_post_review_weight
```

Comment:
Good idea, I haven't thought this timing through so I don't mind changing it.

How does maybe Friday 4pm sound though, it's the end of the week but someone might still be around in case something breaks?

Reply:
Hm, I don't like the idea of things moving around in the queue while admins are working on it.
What things could potentially break here and what would our options be to fix those? If a push would be needed to fix things, I think that's probably out of the question for Friday 4pm.

Reply:
Updated in 2d43a4f37d